### ISSUE TITLE
Screenshots in background not working after upgrade

### ISSUE THREAD

**Issue by alexpunct** [timelyart/Kairos#16]
It seems after I upgraded to the last version that even though it's set to run in the background, when finishing and generating the summary, it's opening the browser in order to take the screenshots (before it was doing that in the background too).

**Comment by timelyart** [comment]
Can you post the summary configuration from the YAML file?

**Comment by alexpunct** [comment]
```yaml
summary:
  # Create email
  email:
    # List of comma separated values
    to: ['nofomopls@gmail.com']
    # List of comma separated values
    # cc: ['my_cc@example.com', 'my_cc_alias@example.com']
    # List of comma separated values
    # Note: not all email providers support bcc (e.g. gmail)
    bcc: []
    subject: 'Alerts Summary - 4h TRI Setups'
    # Use keyword %SUMMARY in your text and/or html to include the Kairos generated summary
    text: '%SUMMARY'
    # Note: you may include <html> and <body> tags but it is not necessary (Kairos will include them if they aren't found)
    html: '<p><strong>Watchlists:</strong> Binance USDT pairs, Binance BTC pairs, Bitmex perpetual pairs (XBT, ETH)<br>
<strong>Alerts timeframe:</strong> 4 hours<br>
<strong>Setups:</strong> El Tangonator, BOT<br>
If you are not familiar with the setups, you can join the community by <a href="https://www.therationalinvestor.com/?ref=F7qfmveLCidGDeDxBh7RUmk" alt="TRI">clicking here</a><br><br>
 %SUMMARY<em>To unsubscribe reply to this email with the text Unsubscribe.</em>'
    # If you just want to send the signals to webhooks (see your config file) or import them as a watchlist, set this to no
    send: yes
    # if you want to shield recipients from seeing each other's email address, set this to yes
    one-mail-per-recipient: yes
```

**Comment by timelyart** [comment]
Can you add: 
```
webdriver:
  # run the webdriver in the background or not (this overrides the setting in the configuration file)
  run-in-backrgound: yes
```

Like this:

```
summary:
  # Create email
  email:
    # List of comma separated values
    to: ['XXX']
    # List of comma separated values
    # cc: ['my_cc@example.com', 'my_cc_alias@example.com']
    # List of comma separated values
    # Note: not all email providers support bcc (e.g. gmail)
    bcc: []
    subject: 'Alerts Summary - 4h TRI Setups'
    # Use keyword %SUMMARY in your text and/or html to include the Kairos generated summary
    text: '%SUMMARY'
    # Note: you may include <html> and <body> tags but it is not necessary (Kairos will include them if they aren't found)
    html: '<p><strong>Watchlists:</strong> Binance USDT pairs, Binance BTC pairs, Bitmex perpetual pairs (XBT, ETH)<br>
<strong>Alerts timeframe:</strong> 4 hours<br>
<strong>Setups:</strong> El Tangonator, BOT<br>
If you are not familiar with the setups, you can join the community by <a href="https://www.therationalinvestor.com/?ref=F7qfmveLCidGDeDxBh7RUmk" alt="TRI">clicking here</a><br><br>
 %SUMMARY<em>To unsubscribe reply to this email with the text Unsubscribe.</em>'
    # If you just want to send the signals to webhooks (see your config file) or import them as a watchlist, set this to no
    send: yes
    # if you want to shield recipients from seeing each other's email address, set this to yes
    one-mail-per-recipient: yes
webdriver:
  # run the webdriver in the background or not (this overrides the setting in the configuration file)
  run-in-backrgound: yes
```
Let me know if that fixes it for you.

**Comment by alexpunct** [comment]
I already have this at the root level

**Comment by timelyart** [comment]
Could you post the ```[webdriver]``` part of the [kairos.cfg](https://github.com/timelyart/Kairos/blob/master/_kairos.cfg) file as well?

**Comment by alexpunct** [comment]
```[webdriver]
; Path to a webdriver on your local machine.
; Download chromedriver at http://chromedriver.chromium.org/downloads but you should be able to use other webdrivers as well,
; e.g. Mozilla Firefox
path = C:\Sites\chromedriver.exe
; resolution to run chrome in (usually this is equal to the resolution of your device)
resolution = 1980,1024
; Run Kairos in the background
run_in_background = false
; Using the clipboard will improve the performance significantly but Kairos will stop running correctly and interfere.
; with other programs when they are the active window. For now, it also requires run_in_background to be set to false.
clipboard = true
; Defines how long chromedriver should wait for a page to load (seconds).
page_load_timeout = 30
; Defines how long chromedriver should wait for finding elements (seconds).
; This is on a lower level than check_if_exists_timeout (below) and should be an equal or higher number than said option.
; 30 seconds should be ample and - unless you have serious issues running the script - you should not change this.
wait_time_implicit = 30
; Defines how long the script should wait finding elements (seconds).
check_if_exists_timeout = 30[webdriver]
; Path to a webdriver on your local machine.
; Download chromedriver at http://chromedriver.chromium.org/downloads but you should be able to use other webdrivers as well,
; e.g. Mozilla Firefox
path = C:\Sites\chromedriver.exe
; resolution to run chrome in (usually this is equal to the resolution of your device)
resolution = 1980,1024
; Run Kairos in the background
run_in_background = false
; Using the clipboard will improve the performance significantly but Kairos will stop running correctly and interfere.
; with other programs when they are the active window. For now, it also requires run_in_background to be set to false.
clipboard = true
; Defines how long chromedriver should wait for a page to load (seconds).
page_load_timeout = 30
; Defines how long chromedriver should wait for finding elements (seconds).
; This is on a lower level than check_if_exists_timeout (below) and should be an equal or higher number than said option.
; 30 seconds should be ample and - unless you have serious issues running the script - you should not change this.
wait_time_implicit = 30
; Defines how long the script should wait finding elements (seconds).
check_if_exists_timeout = 30```

**Comment by timelyart** [comment]
`run_in_background = false`

Can you set that to **true** and let me know if that fixed it?

**Comment by timelyart** [comment]
The fallback to `run_in_background` in kairos,cfg isn't working for the summary part.

**Comment by timelyart** [comment]
Something just came to my mind. Do you have `import: yes` under `watchlist`?
Importing files will always open a window in the foreground as Kairos has to upload through an upload file dialog.

```
  # create a watchlist of summarized signals
  watchlist:
    # Keywords %DATE (yyyymmdd), %TIME (hhmm)
    # Note: will overwrite watchlists of the same name
    name: 'summary_%DATE_%TIME'
    # Import the watchlist into TV
    # Note: existing watchlists of the same name will be deleted
    # Note: this will make use of your clipboard to insert the filename in TV's upload file dialog
    import: yes
    # Attach to summary email
    attach-to-email: yes
```

**Comment by alexpunct** [comment]
I don't. It's set to no

**Comment by timelyart** [comment]
Damn. Okay. Will look further.

**Comment by timelyart** [comment]
Fixed in version [v1.2.4](https://github.com/timelyart/Kairos/releases/tag/v1.2.4)
